maintenance/projects/js-toolkit/packages/liferay-npm-build-tools-common/src/project/index.ts

```diff
@@ -25,6 +25,12 @@ import {VersionInfo} from './types';
 /** A package manager */
 export type PkgManager = 'npm' | 'yarn' | null;
 
+/** Information on the preset being used */
+export interface PresetInfo {
+	name: string;
+	isAutopreset: boolean;
```

Comment:
nit: portal would complain about sorting here... I am less concerned about it, but I'll create [this issue](https://github.com/liferay/liferay-frontend-projects/issues/579) to investigate an eslint plugin to sort this for us.

Reply:
No need to investigate anything: linting is disabled in `maintenance/js-toolkit`, that's why it doesn't complain (I noticed it yesterday). I was thinking in fixing as much as I can, send a PR, and activate it. Otherwise, it will keep drifting and get to a point where it's impossible to fix it.

Reply:
Created [this issue](https://github.com/liferay/liferay-frontend-projects/issues/580) to activate eslint in toolkit v2

Reply:
Not sure about my previous comment. The problem I was seeing was due to be trying to run `eslint` from the project directory when it needs to be run from the root project of `liferay-frontend-projects` because otherwise the configuration are not correctly merged.

However, I still don't understand why this specific rule doesn't kick in. Maybe because it is a type declaration instead of an object? :thinking: I guess it's that.

Sorry for the noise :grimacing:

Reply:
Yeah I think it must be because its a type delcaration. I noticed something similar in Clay awhile back but forgot about it till now